### DISCUSSION TITLE
List of shortcuts

### DISCUSSION THREAD

**Discussion by lindybrito** [xournalpp/xournalpp#5218] [Q&A] [answered]
I'd like to find the list of keyboard shortcuts to xournal++ so i can setup the buttons on my pen and tablet! I already manage to figure out the pen and eraser hahaha.

**Comment by rolandlo** [comment, marked as answer]
I have just assembled a [list of shortcuts](https://github.com/rolandlo/xournalpp-libadwaita-mockup/blob/main/shortcuts/main.blp) defined in Xournal++. It's written in the blueprint file format. If you have [Workbench](https://flathub.org/apps/re.sonny.Workbench) installed you can paste that file into the UI tab and preview it (and use the search functionality to find shortcuts). Or you take a look at this demo:

https://github.com/xournalpp/xournalpp/assets/40485433/18b863eb-2fe1-4ba2-96f6-1a4f6fde26bd

Note that the menu shortcuts can be found in the menu in Xournal++.

**Reply by jdujava** [reply]
It would be great if such a **shortcut list** could be more accessible, either directly from Xournal++, from the [website](https://xournalpp.github.io/ ), or the github page.

I remember there also being shortcut in TeX Editor to essentialy press `OK` button, something like `Ctrl`+`Enter`, but it doesn't seem to work in the nightly build. Not sure if it is a bug or I am misremembering or missing something.
_Edit: when running release-1.2.2 Appimage, the `Ctrl`+`Enter` shortcut works, but for some reason it doesn't work when I manually build from source._

**Comment by lindybrito** [comment]
thank you. I tested some of this shortcuts but they are not working in my tablet button. Only on the stylus pen. Any idea how to solve this?

**Reply by rolandlo** [reply]
Please give some specifics, what OS and what tablet you are using and how you set up the tablet buttons.

**Reply by lindybrito** [reply]
Thanks for the help!! i solve problem. I had to pick the xournal app inside the huion driver and then create the shortcuts.
